src/gui/dialogs/game_stats.cpp

```diff
@@ -67,8 +67,8 @@ unit_const_ptr game_stats::get_leader(const int side)
 
 static std::string controller_name(const team& t)
 {
-	static const std::array<t_string, 3> names {{_("controller^Human"), _("controller^AI"), _("controller^Idle")}};
-	return "<span color='#808080'><small>" + names[t.controller().v] + "</small></span>";
+	static const std::array<t_string, 5> names {{_("controller^Idle"), _("controller^Human"), _("controller^AI"), _("controller^Reserved")}};
```

Comment:
Why 5? 🤔

Reply:
I forgot to adjust it after removing the INVALID enum value.  Is there a reason to not do just `return values_.size();` though?